### PR TITLE
Suggestions for readmes

### PR DESCRIPTION
**Pull request by saleandro**
Rephrasing instructions to make more clear what needs to be done for set-up or not.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,11 +42,19 @@ script/{your-framework}/start
...
 
-## Task
+## Your set-up task
 1. Run Foo at http://127.0.0.1:8008, verify that an alert with `Yay! Everything works` fires
 2. Run Bar at http://127.0.0.1:8009, verify that an alert with `Yay! Everything works` fires
 
-## Structure
+🥳 If those worked, well done! You're all set for the interview! 
+
+🤔 If not, something might be off... but don't worry, this is not part of a test! We'll be able to help you set up on the day of the interview.
```
Comment (saleandro) [inline README.md:51]: Is this a fair assumption?

File: README.md [hunk 2]
```diff
@@ -42,11 +42,19 @@ script/{your-framework}/start
 
 This will get you the foo site running at http://127.0.0.1:8008, the bar interface running at http://127.0.0.1:8009 and a webserver running at http://127.0.0.1:3000.
 
-## Task
+## Your set-up task
 1. Run Foo at http://127.0.0.1:8008, verify that an alert with `Yay! Everything works` fires
 2. Run Bar at http://127.0.0.1:8009, verify that an alert with `Yay! Everything works` fires
 
-## Structure
+🥳 If those worked, well done! You're all set for the interview!
```
Comment (saleandro) [inline README.md:49]: Making it clear that they don't need to do anything else for the set-up - can someone confirm that's correct?

File: README.md [hunk 3]
```diff
@@ -58,15 +66,11 @@ Your framework will have a folder with it's name and have 2 endpoints:
...
 
-Each Server framework is in its own folder (named after the framework) and has its own readme. You can safely ignore anything in the frameworks that you are not using.
+Each Server framework is in its own folder (named after the framework) and has its own Readme. You can safely ignore anything in the frameworks that you are not using.
 
 - [Rails](./rails/README.md)
 - [Spring](./spring/README.md)
 - [Django](./django/README.md)
 - [Node](./node/README.md)
-
-## Prototype!
```
Comment (saleandro) [inline README.md:70]: I removed this bit as it confused me during the set-up and I don't feel it's relevant then

File: django/README.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
 # Django
 
-Welcome to the Django backend! For all operations run them from the project root folder.
+Welcome to the Django backend! 
+
+There's nothing to be done here for the interview set-up, but feel free to look around! 👀
```
Comment (saleandro) [inline django/README.md:5]: I added this phrase to all frameworks' readmes - confirm if this is the case

File: README.md (hunk 1 above)
Comment (pratik60) [reply README.md:51]: I'm not sure we would want to phrase it that way. I think we want to make sure it's up and working for you, otherwise on the day of the test it's unlikely we will have this setup in 15 minutes which has caused interviews to run over, postpone to next day, or damaged the person's setup as we tried one command after another. 

cc @patocallaghan to confirm

Comment (saleandro) [reply README.md:51]: Got it! What else can we tell them to help them debug?

Comment (saleandro) [reply README.md:51]: Or should they let us know ahead of the interview that it hasn't worked?

Comment (patocallaghan) [reply README.md:51]: Generally the problem this repo is trying to solve is that people can't get all of their dependencies set up or the servers running on the day. We imagine some folks will have some issues due to how their local environments are set up. I guess they should be making sure the setup, start, foo and bar scripts ran without any errors.

Comment (saleandro) [reply README.md:51]: I understand that that's the ideal goal - I'm just wondering, if I'm a candidate, and I can't get this working, what can I do about that? 

We could tell candidates one or some of these:
- Say nothing - and on the day of the interview we get a worried candidate without a working set-up 😞
- Give them guidance on how to fix those errors, maybe there are common error cases we know about? 
- Tell them our expectation that they should figure out themselves how to set up this project - is that the case?
- Ask them to send us feedback on how this setup process failed for them - maybe at least for the first few candidates?

But ultimatley, I think we should reassure them that the interview will still go ahead (right?!) even if they have issues - and that we are here to help them be successful. For that, we can either provide support before the interview or on the interview day.

Does that make sense? Happy to chat if easier!

Comment (patocallaghan) [reply README.md:51]: 👍 I'd say it would be a combo of 

* > Give them guidance on how to fix those errors, maybe there are common error cases we know about
* > Ask them to send us feedback on how this setup process failed for them - maybe at least for the first few candidates?`.

We mention in the email that we send to candidates to let us know if they couldn't get set up with the repo but no harm having it here too. If they have trouble setting up we may be able to schedule more time for setup on the day rather than the usual 15 mins.

Comment (saleandro) [reply README.md:51]: Thanks for the context Pat! What if I rephrase it to:
```suggestion
🤔 If not, please let us know ahead of the interview. We want to make sure you have it all set up before the interview day.
```

Comment (patocallaghan) [reply README.md:51]: I like it 👌

Comment (pratik60) [reply README.md:51]: That rephrasing sounds great Sabrina! 👍
